torch/csrc/jit/tensorexpr/operators/misc.cpp

```diff
@@ -382,12 +382,11 @@ Tensor computeExpand(
       });
 }
 
-static Tensor computeReshapeHelper(
+Tensor computeReshape(
     const std::vector<ArgValue>& inputs,
     const std::vector<ExprHandle>& outputShape,
     const c10::optional<ScalarType>& outputType,
-    at::Device device,
-    const IntList& view_dims) {
+    at::Device device) {
   auto A = c10::get<BufHandle>(inputs[0]);
   if (A.ndim() == 0) {
     return Compute(
```

Comment:
This is not supporting `aten::view` in the general form. How are we restricting the cases we handle with `view`? 

Probably not directly relevant to this PR, but I just noticed this.

Reply:
That's right, we're modeling `view` as `reshape`, which is not correct in a general case. We don't accept `view` in the fuser though, so this only is used in AOT scenario where users need to opt in.